snuba/cli/api.py

```diff
@@ -46,13 +47,11 @@ def api(
         WSGIRequestHandler.protocol_version = "HTTP/1.1"
         application.run(host=host, port=port, threaded=True, debug=debug)
     else:
-        import mywsgi
-
         if log_level:
             os.environ["LOG_LEVEL"] = log_level
 
         with flush_attribution_producer():
```

Comment:
unrelated to this patch but this call does nothing!  both `myuwsgi.run` and `snuba.utils.uwsgi.run` end with `os.execvp` which replaces the current process